bpf/mock/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM docker.io/library/ubuntu:20.04
+RUN apt-get update
+RUN apt-get -y install build-essential
+RUN apt-get -y install ruby-full
+RUN apt-get -y install git
+RUN git clone --recursive https://github.com/ThrowTheSwitch/CMock.git
+RUN git -C CMock reset --hard 3d4ba8d20b8958da5dace7dd5d31155c94b60819
```

Comment:
:thinking: do we still need two different dockerfiles to facilitate the overall process?

Even if we need to run docker multiple times, if the image is the same (as `test/bpf/Dockerfile`) then is it possible to add just one dockerfile and share the build / tag steps?

Reply:
That makes a lot of sense. Addressed!